docs/src/en/user-guide/apm/send-traces-to-holoinsight.md

```diff
@@ -0,0 +1,51 @@
+# Send Traces to HoloInsight
+> Before this you need to [deploy the HoloInsight collector](../../operations/deployment/deploy-holoinsight-collector.md).
+
+Report the trace data to the HoloInsight collector for processing and export to the HoloInsight server, and then you can view the distributed trace links, service topology, and aggregated statistical indicators on the HoloInsight dashboard.
+
+The HoloInsight collector receives trace data based on the [OpenTelemetry](https://opentelemetry.io/) specification, which means that you can use almost any open-source SDK (OTel, SkyWalking, etc.) to collect data and report it uniformly, depending on your software environment and programming language.
+
+# How to configure
+
+> The `YOUR_APPLICATION_NAME` mentioned below is used to identify the `application` (aka the `service`), which should be the same as the application name specified by `-a` when installing HoloInsight Agent.
+
+## Using OTel SDK
+> To be added.
+
+## Using SkyWalking Agent
+### If your application is deployed in a container
+1. Edit the Dockerfile to download [SkyWalking Agent](https://skywalking.apache.org/downloads/#Agents) and configure
+```
+# Download and unzip the SkyWalking Agent
+RUN wget https://archive.apache.org/dist/skywalking/java-agent/8.11.0/apache-skywalking-java-agent-8.11.0.tgz
```

Comment:
update skywalking java agent 8.11.0 to  8.15.0

Reply:
fixed.